PHP/PHP.sublime-completions

```diff
@@ -5478,7 +5478,7 @@
 			"trigger": "ftp_login()",
 			"contents": "ftp_login(${1:ftp_stream}, ${2:username}, ${3:password})",
 			"kind": "function",
-			"details": "Logs in to an FTP connection"
+			"details": "Logs in to an FTP server"
```

Comment:
Is this correct?

Reply:
I believe so, you don't _log in_ to a connection. You do _log in_ to a server or service running on a server. You might _open_ a connection.